extension/package.json

```diff
@@ -13,13 +13,13 @@
     "lint:fix": "eslint --ext .js,.vue src --fix && stylelint \"src/**/*.{css,vue}\" --fix",
     "lint:js": "eslint --ext .js,.vue src",
     "lint:css": "stylelint \"src/**/*.{css,vue}\"",
-    "build": "node ./scripts/check-env-variables.js && cross-env NODE_ENV=production webpack --hide-modules && node ./scripts/remove-localhost.js",
-    "build:dev": "node ./scripts/check-env-variables.js && cross-env NODE_ENV=development webpack --hide-modules",
+    "build": "del dist && node ./scripts/check-env-variables.js && cross-env NODE_ENV=production webpack --hide-modules && node ./scripts/remove-localhost.js",
```

Comment:
what is this command? delete? Does this also work in Linux? (`rm -r`)

Reply:
this is an npm package to work cross platform